config/features.yml

```diff
@@ -908,3 +908,6 @@ features:
     actor_type: user
     description: Use lighthouse instead of EVSS to view benefit claims
     enable_in_development: true
+  virtual_agent_lighthouse_claims:
+    actor_type: user
+    description: Use lighthouse instead of EVSS to view benefit claims
```

Comment:
@anitadewitt-va This is the same description as the feature toggle above it 😕. Can you just use that toggle instead? If not, can you make this description more specific?

Reply:
I can make the description more specific. Our team does not want to interfere with what the creator's of that feature toggle are doing. We will need to turn our toggle on and off independent of their usage.